asapdiscovery-modeling/asapdiscovery/modeling/scripts/slice_fragalysis_structures_with_mdtraj.py

```diff
@@ -0,0 +1,131 @@
+"""
+This script slices the structures from the fragalysis database into the active site and full protein PDB files,
+as well as the corresponding numpy arrays of the coordinates of the atoms in the active site and full protein.
```

Comment:
maybe I'm missing something but I'm still not exactly sure what this script does? does it just extract just-protein and just-active-site from a PDB file?

Reply:
yes, its probability overkill for this case but could be nice to have for simulations